### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by wacraven**
completed quiz. corrected return key event listener

### CODE REVIEW

File: quiz.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+document.getElementById("treeHeight").addEventListener("keydown", enterListener);
+
+function enterListener(key) {
+	if (13 == key.keyCode) {
+		tree()
+	}
+}
+
+
+function tree() {
```
Comment (stevebrownlee) [inline quiz.js]: **Requirement not met:** Create a tree function that should build a pine tree out of a character in the Chrome dev tools console. It accepts a single object as an argument.

**Comment by stevebrownlee** [comment]
@wacraven Great job getting the tree working. Unfortunately, you missed the first requirement. Need to refactor to ensure you meet all requirements.  Also, type in `-1` into the size of the tree field and see what happens. Make sure you account for that edge case in your code as well.

Ping Joe on Slack when you're ready for another review.

File: quiz.js [hunk 2]
```diff
@@ -1,25 +1,38 @@
...
+function clickListener() {
+	tree(treeObject);
+}
+
+/*****************TREE BUILD**************************/
+
+function tree(obj) {
+	obj.hgt = Math.abs(document.getElementById('treeHeight').value);
+	obj.char = document.getElementById('character').value;
+	if (obj.hgt == 0) {
```
Comment (JoeShep) [inline quiz.js]: Missing a bock of code here. I'm guessing you're dealing with the edge case of a height that's 0 or a negative number? If so, this is a good place to throw an error or some kind to let your user know they need to add a correct input.

**Comment by JoeShep** [comment]
💥 There you go. Looks good. You are ready to merge

**Pull request merged**